pkg/infrastructure/gcp/clusterapi/operation.go

```diff
@@ -0,0 +1,47 @@
+package clusterapi
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"google.golang.org/api/compute/v1"
+)
+
+// WaitForOperationGlobal will attempt to wait for a operation to complete where the operational
+// resource is globally scoped.
+func WaitForOperationGlobal(ctx context.Context, projectID string, operation *compute.Operation) error {
+	ctx, cancel := context.WithTimeout(ctx, time.Minute*1)
+	defer cancel()
+
+	service, err := NewComputeService()
+	if err != nil {
+		return fmt.Errorf("failed to create compute service: %w", err)
```

Comment:
Nit - could just return `err` since NewComputeService() is returning same message